autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/CloudbreakCommunicator.java

```diff
@@ -36,13 +36,13 @@ public StackV4Response getByCrn(String stackCrn) {
     }
 
     public AutoscaleStackV4Response getAutoscaleClusterByCrn(String stackCrn) {
-        return cloudbreakInternalCrnClient.withUserCrn(restRequestThreadLocalService.getCloudbreakUser().getUserCrn())
+        return cloudbreakInternalCrnClient.withInternalCrn()
                 .autoscaleEndpoint().getAutoscaleClusterByCrn(stackCrn);
     }
 
-    public AutoscaleStackV4Response getAutoscaleClusterByName(String stackName) {
-        return cloudbreakInternalCrnClient.withUserCrn(restRequestThreadLocalService.getCloudbreakUser().getUserCrn())
-                .autoscaleEndpoint().getAutoscaleClusterByName(stackName);
+    public AutoscaleStackV4Response getAutoscaleClusterByName(String accountId, String stackName) {
+        return cloudbreakInternalCrnClient.withInternalCrn()
+                .autoscaleEndpoint().getInternalAutoscaleClusterByName(accountId, stackName);
     }
```

Comment:
accountId and stackName interchanged compared to the endpoint.

Reply:
good catch, thanks!